libpod/runtime.go

```diff
@@ -565,6 +566,31 @@ func (r *Runtime) Info() ([]InfoData, error) {
 	return info, nil
 }
 
+// GenerateName generates a unigue name for a container or pod.
+func (r *Runtime) GenerateName() (string, error) {
```

Comment:
Can probably be private, should never need this outside of libpod

Reply:
Fixed